recipes/fluiddyn/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "fluiddyn" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 488dcc8cd1660fd6eed939defeb21250114b5de9c94cc1d979136ea439789b33
+
+build:
+  number: 0
+  entry_points:
+    - fluidinfo = fluiddyn.util.info:main
+    - fluidnbstripout = fluiddoc.fluidnbstripout:main
+    - fluiddocset = fluiddoc.fluiddocset:main
+    - fluiddump = fluiddyn.io.dump:main
+    - fluidmat2py = fluiddyn.util.matlab2py:main
+    - fluidcluster-help = fluiddyn.clusters:print_help_scheduler
+  script: "{{ PYTHON }} -m pip install .[full] --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
```

I think this can be noarch?
```suggestion
  noarch: python
```